src/Tickets/Commerce/Gateways/Square/Webhooks.php

```diff
@@ -265,6 +264,30 @@ public function verify_signature( $received_secret_key ) {
 		return wp_check_password( $unhashed_key, $received_secret_key );
 	}
 
+	/**
+	 * Verify whodat signature.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $payload       The payload from the request.
+	 * @param string $received_hash The hash from the request.
+	 *
+	 * @return bool Whether the signature is valid.
+	 */
+	public function verify_whodat_signature( string $payload, string $received_hash ) {
+		$signature        = tribe( Merchant::class )->get_whodat_signature();
+		$notification_url = $this->get_webhook_endpoint_url();
+
+		if ( empty( $signature ) || empty( $notification_url ) || empty( $payload ) ) {
```

Comment:
No need for empty check - we can just do 
```suggestion
		if ( ! ( $signature && $notification_url && $payload ) ) {
```